docs/authoring-recipes/recipe-development-environment.md

```diff
@@ -55,6 +55,10 @@ mvn -B archetype:generate -DgroupId=com.mycompany.app -DartifactId=my-app -Darch
 </TabItem>
 </Tabs>
 
+:::info
+If you are using Gradle, it is highly-recommended that you apply the [`org.openrewrite.build.recipe-library-base` plugin](https://plugins.gradle.org/plugin/org.openrewrite.build.recipe-library-base). The plugin automatically configures the project with meaningful conventions like necessary compiler options.
```

Comment:
The plugin hadn't been mentioned yet but I think it's super important to always go with it.